.travis.yml

```diff
@@ -12,7 +12,17 @@ env:
   - TOXENV=py33-1.8
   - TOXENV=py34-1.8
   - TOXENV=py34-1.9
+  - TOXENV=py34-1.10
+  - TOXENV=py34-1.11
   - TOXENV=py34-master
+  - TOXENV=py35-1.9
+  - TOXENV=py35-1.10
+  - TOXENV=py35-1.11
```

Comment:
I'm not saying it doesn't **support** them, I'm saying it doesn't **ship** with them by default. You need that extra directive.